pkg/action/install.go

```diff
@@ -410,14 +403,10 @@ func (i *Install) failRelease(rel *release.Release, err error) (*release.Release
 //	- used by a deleted release, and i.Replace is false
 func (i *Install) availableName() error {
 	start := i.ReleaseName
-	if start == "" {
-		return errors.New("name is required")
-	}
 
-	if len(start) > releaseNameMaxLen {
-		return errors.Errorf("release name %q exceeds max length of %d", start, releaseNameMaxLen)
+	if err := chartutil.ValidateReleaseName(start); err != nil {
+		return errors.Errorf("install: Release name is invalid: %s, err: %s", start, err)
```

Comment:
Can you try changing the error return:
return errors.Errorf("install: Release name is invalid: %s, err: %s", start, err)

to the following: return errors.Wrapf(err, "release name %q is invalid", start)

It would follow how we wrap error strings usually.

Reply:
Thanks for your advice, it has been updated.

Reply:
Looking at the error returned again during manual testing, do you mind changing to the following as it would read better:

return errors.Wrapf(err, "release name %q", start)

Reply:
@mengjiao-liu  can you remove "is invalid""? ^^

Reply:
Ok, updated.